handlers/ExecuteBlockActionHandler.ts

```diff
@@ -75,6 +75,81 @@ export class ExecuteBlockActionHandler {
                 }
                 break;
             }
+            case Modals.COMMENT_INPUT_ACTION: {
+                if (value) {
+                    const association = new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, `regen_input`);
+                    await this.persistence.updateByAssociation(association, { regen_input: value }, true);
+                }
+                break;
+            }
+            case Modals.GEN_INPUT_ACTION: {
+                if (value) {
+                    const association = new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, `gen_input`);
```

Comment:
if we are using `gen_input` in multiple places we should store it in a const variable.